examples/with-rollup/packages/ui/package.json

```diff
@@ -1,23 +1,31 @@
 {
-  "name": "ui",
+  "name": "@repo/ui",
   "version": "0.0.0",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "license": "MIT",
+  "type": "module",
+  "exports": {
+    "./button": {
+      "types": "./dist/Button.d.ts",
+      "default": "./dist/button.js"
+    },
+    "./header": {
+      "types": "./dist/Header.d.ts",
+      "default": "./dist/header.js"
+    }
+  },
   "scripts": {
     "lint": "eslint *.ts*",
     "build": "rollup --config",
     "dev": "pnpm run build -- --watch"
   },
   "devDependencies": {
-    "@rollup/plugin-typescript": "^8.5.0",
-    "@types/react": "^18.2.5",
-    "@types/react-dom": "^18.2.4",
-    "eslint": "^7.32.0",
-    "eslint-config-custom": "workspace:*",
+    "@repo/eslint-config": "workspace:*",
+    "@repo/typescript-config": "workspace:*",
+    "@rollup/plugin-typescript": "^11.1.5",
+    "@types/react": "^18.2.38",
+    "@types/react-dom": "^18.2.17",
+    "eslint": "^8.54.0",
```

Comment:
Move eslint to root

Reply:
I've always operated as, conceptually, a package should be as hermetic as possible?

Reply:
@anthonyshew generally yes, for anything needed to build / run. But for dev requirements root is ok. We do this with prettier currently. It's also nicer for Eslint because version drift can cause mayhem and it's easy to do accidentally. So keeping one single version at root is a much nicer dev experience with no drawbacks.